devicetypes/rachio/rachio-iro2-zone.src/rachio-iro2-zone.groovy

```diff
@@ -86,84 +36,14 @@ metadata {
     tiles (scale: 2){
         multiAttributeTile(name: "valveTile", type: "generic", width: 6, height: 4) {
             tileAttribute("device.watering", key: "PRIMARY_CONTROL" ) {
-                attributeState "off", label: 'Off', action: "open", icon: "st.valves.water.closed", backgroundColor: "#ffffff", nextState:"updating"
+                attributeState "off", label: 'Off', action: "open", icon: "st.valves.water.closed", backgroundColor: "#ffffff", nextState:"on"
                 attributeState "offline", label: 'Offline', icon: "st.valves.water.closed", backgroundColor: "#cccccc"
```

Comment:
If we are changing to the "switch" capability then "offline" is not a valid value